.markdownlint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "default": true,
+  "MD004": false,
+  "MD013": false,
```

Comment:
nit / opinion - the descriptive names (e.g., "line-length" in this case) are easier to read / understand. Using the rule IDs requires opening the map.